Enable Flask remote debugging i Docker

ZhuXiaohua11 opened the pull request:
Added port 5678 mapping for remote debugging with debugpy

Updated command to run Flask through debugpy

Kept port 5000 for normal app access

ZhuXiaohua11 committed:
add .env i docker-compose.yml

ZhuXiaohua11 committed:
add debugpy to enable debug i docker

dangosailing commented:
routes/assistant_routes.py and  controllers/assistant_controller.py do the same thing? Think assistant_controller.py might have been added by mistake?

ZhuXiaohua11 committed:
Delete backend/src/controllers/assistant_controller.py

dangosailing reviewed (changes requested):
About debugpy:
It needs to go in requirements.in, not directly into requirements.txt
Remove it from requirements.txt and add it to requirements.in
Then run pip-compile to add it to requirements.txt

ZhuXiaohua11 commented:
> About debugpy: It needs to go in requirements.in, not directly into requirements.txt Remove it from requirements.txt and add it to requirements.in Then run pip-compile to add it to requirements.txt

Good catch! You’re right — controllers/assistant_controller.py was accidentally added. I have removed it and tested in local ,updated the PR. Thanks!

ZhuXiaohua11 committed:
uppdate requirements.in/.txt

dangosailing commented:
Important, don't upload env variables!!! Don't commit actual values like api key to github.
The env examples file is only for placeholder variable names. Like so: 
DB_NAME=

ZhuXiaohua11 commented:
> About debugpy: It needs to go in requirements.in, not directly into requirements.txt Remove it from requirements.txt and add it to requirements.in Then run pip-compile to add it to requirements.txt

Done

ZhuXiaohua11 committed:
env sample deleted

ZhuXiaohua11 commented:
> Important, don't upload env variables!!! Don't commit actual values like api key to github. The env examples file is only for placeholder variable names. Like so: DB_NAME=

Done 👍

ZhuXiaohua11 commented:
Enable Flask remote debugging i Docker
